### ISSUE TITLE
Check out my app tradebindr

### ISSUE THREAD

**Issue by tomreece** [chasestarr/mtg_binder#1]
https://github.com/tomreece/tradebindr

www.tradebindr.com

Wanna work together to build something?

**Comment by chasestarr** [comment]
Yeah! I'm interested in working together. I've been thinking about doing something similar with a localized trading app. 

Take a look at this idea dump that I made a couple weeks ago on the topic. https://github.com/chasestarr/ideas/issues/1
